tutorials/group_level_analysis/plot_group_level_with_individual_clusters.py

```diff
@@ -6,68 +6,71 @@
-#     environment:
+#     The lemon datasets used in this tutorial is composed of EEGLAB files. To
+#     use the MNE reader :func:`mne.io.read_raw_eeglab`, the ``pymatreader``
+#     optional dependency is required. Use the following installation method
+#     appropriate for your environment:
 #
 #     - ``pip install pymatreader``
 #     - ``conda install -c conda-forge pymatreader``
 #
-#     Note that an environment created via the MNE installers includes
+#     Note that an environment created via the `MNE installers`_ includes
 #     ``pymatreader`` by default.
 
+# sphinx_gallery_thumbnail_number = 2
+
+import numpy as np
 from mne.io import read_raw_eeglab
 
 from pycrostates.cluster import ModKMeans
 from pycrostates.datasets import lemon
-from pycrostates.preprocessing import extract_gfp_peaks
 from pycrostates.io import ChData
+from pycrostates.preprocessing import extract_gfp_peaks
+
 
 condition = "EO"
 subject_ids = ["010020", "010021", "010022", "010023", "010024"]
 
 #%%
-# In this example, we first compute individual
-# topographies from GFP peaks (subject level analysis).
-# Then we concatenate individual topographies into
-# a single dataset and submit it for a second round of
-# clustering (group level analysis).
-
-import numpy as np
-
-n_jobs = 2
+# In this example, we start with a subject level analysis by computing
+# individual topographies from :term:`GFP` peaks. Then, the individual
+# topographies are concatenated and used in the group level analysis to fit a
+# second clustering algorithm.
 
 individual_cluster_centers = list()
 for subject_id in subject_ids:
-    # Load Data
-    ModK = ModKMeans(n_clusters=5, random_state=42)
+    # load Data
     raw_fname = lemon.data_path(subject_id=subject_id, condition=condition)
     raw = read_raw_eeglab(raw_fname, preload=True)
     raw = lemon.standardize(raw)
     raw.pick("eeg")
-    # For sake of time, we only use 30s of recording.
-    raw.crop(0, 30)
+    raw.crop(0, 30)  # crop the dataset to speed up computation
     raw.set_eeg_reference("average")
-    # Extract Gfp peaks
+    # extract GFP peaks
     gfp_peaks = extract_gfp_peaks(raw)
-    # Subject level clustering
-    ModK.fit(gfp_peaks, n_jobs=n_jobs)
-    individual_cluster_centers.append(ModK.cluster_centers_.T)
+    # subject level clustering
+    ModK = ModKMeans(n_clusters=5, random_state=42)
+    ModK.fit(gfp_peaks, n_jobs=2)
+    individual_cluster_centers.append(ModK.cluster_centers_)
 
-group_cluster_centers = np.hstack(individual_cluster_centers)
+group_cluster_centers = np.vstack(individual_cluster_centers).T
 group_cluster_centers = ChData(group_cluster_centers, ModK.info)
 
-# Group level clustering
+# group level clustering
 ModK = ModKMeans(n_clusters=5, random_state=42)
-ModK.fit(group_cluster_centers, n_jobs=n_jobs)
+ModK.fit(group_cluster_centers, n_jobs=2)
 ModK.plot()
 
 #%%
-# We can reorganize our clustering results to our needs.
```

Comment:
For the end of this tutorial, instead of importing pandas and creating a DataFrame, could you remove the import and instead create a couple of plots about those metrics? It would be more visual. @vferat

Reply:
I removed pandas and added a violinplot of the GEV explained by each MS